python/pyproject.toml

```diff
@@ -96,8 +96,6 @@ test = [
 ]
 
 all = [
-    "aac[runtime]",
```

Comment:
i think you have to list dependencies here as well so that it gets included with the others?

Reply:
I don't think it's necessary to include them in both places.

Reply:
Yeah, I just dug into the install step in the pipeline and looks like it does properly grab them, so should be good to go.